File: UIPreparation/UI/src/app/core/components/admin/warehouse/models/warehouseDto.ts
export class WarehouseDto {
  id?: number;
  createdUserId?: number;
  createdDate?: Date | any;
  userName?:string;
  lastUpdatedUserId?: number;
  lastUpdatedDate?: Date | any;
  status: boolean;
  isDeleted: boolean;
  productId?: number;
  productName?: string;
  size?: string;
  color?: string;
  quantity?: number;
  readyForSale?: string;
}

File: Entities/Dtos/WarehouseDto.cs
using Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Dtos
{
    public class WarehouseDto : BaseEntityDto, IDto
    {
        
        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public string Color { get; set; }
        public string UserName { get; set; }
        public int Quantity { get; set; }
        public string ReadyForSale { get; set; }
        public string Size { get; set; }
    }
}
